lib/hammer_cli/modules.rb

```diff
@@ -76,14 +76,45 @@ def self.require_module(name)
     end
 
     def self.load_all
-      HammerCLI::Modules.names.each do |m|
+      module_names = HammerCLI::Modules.names
+      module_names.each do |m|
         Modules.load(m)
       end
       loaded_for_deps = loaded_modules & disabled_modules
       unless loaded_for_deps.empty?
         message = _("Error: Some of the required modules are disabled in configuration: %s.") % loaded_for_deps.join(', ')
         raise HammerCLI::ModuleDisabledButRequired.new(message)
       end
+      return if modules_checksum
+
+      File.open(checksum_file_path, 'w') do |f|
+        f.write(compute_modules_checksum(module_names))
+      end
```

Comment:
There's a shorthand for open-write-close
```suggestion
      File.write(checksum_file_path, compute_modules_checksum(module_names))
```